### PR TITLE
nmstatectl: Verify state by default

### PR DESCRIPTION
**Pull request by tyll**
This is another split from #73

### CODE REVIEW

**Comment by EdDev** [comment]
I think we need to update the documentation as well.

**Comment by tyll** [comment]
The documentation does not seem to mention `--verify` - also checked the other repo with examples. Only the demo will not be accurate anymore.

**Comment by tyll** [comment]
To test the network system role I created veth devices to have some kind of dummy devices. Maybe this can be done here as well?

**Comment by EdDev** [comment]
In the docker env, eth0 is actually a veth and we should be able to create several of them when running the container.
I just thought to create an iface by defining it.. a dummy is a good candidate as we are supposed to support dummy type, unfortunately it does not work well on first try (probably we need the wait-for-connection-active mechanism).

File: nmstatectl/nmstatectl.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def setup_subcommand_set(subparsers):
                             'stdin is used when no file is specified.',
                             nargs='?')
     parser_set.add_argument(
-        '--verify', action='store_true',
+        '--no-verify', action='store_false', dest='verify', default=True,
         help='Verify that the desired state is now part of the current state.'
```
Comment (cathay4t) [inline nmstatectl/nmstatectl.py]: The help text seems does not explain `no-verify`. Maybe something like: `Disable the verify of desired state is now part of the current state.`

Comment (EdDev) [reply nmstatectl/nmstatectl.py]: I would suggest: `Do not verify that the desired state is part of the current state post execution.`

Comment (tyll) [reply nmstatectl/nmstatectl.py]: Good catch. I changed it to:

`Do not verify that the state was completely set and disable rollback to previous state.`

How do you like it?

**Pull request merged**